test/Psr/CacheItemPool/CacheItemPoolDecoratorTest.php

```diff
@@ -295,6 +301,9 @@ public function testGetItemsRuntimeExceptionIsMiss(): void
             ->with($keys)
             ->willThrowException(new Exception\RuntimeException());
 
+        /**
+         * @var array<string, CacheItem> $items
```

Comment:
do we have to provide these here?

Reply:
Yes because psalm is complaining that can not understand data, since response is "iterable"

Reply:
Yes, psalm does not work because original documentation inside interface gives just return type iterable and it is not enough. I tried to add it on implemented methods but psalm does not work with 2 return types defined (did not want to loose original doc type)